.github/workflows/dependencies.yaml

```diff
@@ -33,8 +33,13 @@ jobs:
         sed -i "/^K0SCTL_VERSION/c\K0SCTL_VERSION = $VERSION" Makefile
     - name: K0s
       run: |
+        # Remove the '-ec.X' suffix and only update if the prefix (upstream k0s release) has changed.
+        export CURVERSION=$(grep ^K0S_VERSION Makefile | cut -d= -f2 | tr -d " " | cut -d - -f1)
```

Comment:
This could be simplified to: `awk '/^K0S_VERSION/{split($3,a,"-"); print a[1]}' Makefile`

Reply:
Added.